test/test_party_controller.py

```diff
@@ -336,16 +336,16 @@ def test_get_party_with_nonexistent_ref(self):
         self.get_party_by_ref('B', '123', 404)
 
     def test_get_party_by_survey_id_and_enrolment_statuses_with_valid_enrolment(self):
-        self.populate_with_respondent(respondent=self.mock_respondent_with_id)
+        self.populate_with_respondent(respondent=self.mock_respondent_with_id)  # NOQA
         mock_business = MockBusiness() \
             .attributes(source='test_get_business_by_id_returns_correct_representation') \
             .as_business()
         mock_business['id'] = '3b136c4b-7a14-4904-9e01-13364dd7b972'
         self.post_to_businesses(mock_business, 200)
         self._make_business_attributes_active(mock_business=mock_business)
         self.associate_business_and_respondent(business_id=mock_business['id'],
-                                               respondent_id=self.mock_respondent_with_id['id'])
-        self.populate_with_enrolment()
+                                               respondent_id=self.mock_respondent_with_id['id'])  # NOQA
+        self.populate_with_enrolment()  # NOQA
```

Comment:
why were you getting linting errors for these

Reply:
Due to the use of the db_session decorator it complains about no value for session

Reply:
oh this is codacy? yeah its a bitch